app/livechat/client/views/app/tabbar/customerChatHistory.js

```diff
@@ -0,0 +1,148 @@
+import { Template } from 'meteor/templating';
+import moment from 'moment';
+import { ReactiveVar } from 'meteor/reactive-var';
+import './customerChatHistory.html';
+import './chatRoomHistoryItem.html';
+import './chatRoomSearchItem.html';
+import _ from 'underscore';
+
+import { APIClient } from '../../../../../utils/client';
+
+const ITEMS_COUNT = 50;
+Template.customerChatHistory.helpers({
+
+	hasChatHistory() {
+		return Template.instance().hasHistory.get();
+	},
+	isSearching() {
+		return Template.instance().isSearching.get();
+	},
+	isAllChat() {
+		return Template.instance().isAllChat.get();
+	},
+	isChatClicked() {
+		return Template.instance().isChatClicked.get();
+	},
+	isfound() {
+		return Template.instance().isFound.get();
+	},
+	searchResults() {
+		return 	Template.instance().searchResult.get();
+	},
+	previousChats() {
+		return Template.instance().history.get();
+	},
+	clickRid() {
+		return Template.instance().clickRid.get();
+	},
+	clickToken() {
+		return Template.instance().clickToken.get();
+	},
+	title() {
+		let title = moment(this.ts).format('L LTS');
+
+		if (this.label) {
+			title += ` - ${ this.label }`;
+		}
+		return title;
+	},
+});
+
+const DEBOUNCE_TIME_FOR_SEARCH_DEPARTMENTS_IN_MS = 300;
```

Comment:
Why not using a correct name for this CONST?
DEBOUNCE_TIME_FOR_SEARCH_DEPARTMENTS_IN_MS -> DEBOUNCE_TIME_FOR_SEARCH_CHATS_IN_MS

Reply:
done